devenv/docker/blocks/mysql/docker-compose.yaml

```diff
@@ -1,5 +1,5 @@
   mysql:
-    image: mysql:${mysql_version}
+    image: arm64v8/mysql:oracle # ${mysql_version}
```

Comment:
this won't be merged.... but is how I can get mysql running on an m1

Reply:
approving, but just wanted to add a reminder about this one